R/utilities-molecule-plots.R

```diff
@@ -22,7 +22,11 @@
         color = .data[[mapLabels$color]],
         shape = .data[[mapLabels$shape]],
         alpha = if (mapLabels$lloq != "legendLabels") {
-          (.data[[mapLabels$y]] > .data[[mapLabels$lloq]])|(.data[[mapLabels$x]] > .data[[mapLabels$lloq]])
+          switch (plotObject$plotConfiguration$lloqDirection,
+            "horizontal" = (.data[[mapLabels$y]] < .data[[mapLabels$lloq]]),
+            "vertical" =  (.data[[mapLabels$x]] < .data[[mapLabels$lloq]]),
+            "both" = ((.data[[mapLabels$y]] < .data[[mapLabels$lloq]])|(.data[[mapLabels$x]] < .data[[mapLabels$lloq]]))
```

Comment:
should  ("horizontal", "vertical", "both") be defined as a new enum (`LloqDirection `or so)?

Reply:
We also have the error bars that use `directions` so maybe a `Direction` Enum ?